### PR TITLE
ACMS-754:Update every ACMS module's composer.json with required dependencies

### PR DESCRIPTION
**Pull request by saurabhtripathics**
https://backlog.acquia.com/browse/ACMS-754

https://docs.google.com/document/d/1osse6r2byRc5hoUQ-s09Y2gXGu4TnU_1FF9wLDNz6ik/edit#

**Motivation**
Update every acms module's composer.json with required dependencies.

**Proposed changes**
Update every acms module's composer.json with required dependencies.
 

**Testing steps**
Follow ACMS-618, where we have reps created for module. We need to test if the repo's composer.json is properly installing the dependencies.

### CODE REVIEW

**Comment by chandan-singh7929** [comment]
@saurabhtripathics - I think this PR should be created against `feature/profile-refactor` branch instead of develop.
cc: @vishalkhode1

**Comment by saurabhtripathics** [comment]
@vishalkhode1  do let me know if base is correct, as whatever base we choose, we may have to rebase.

File: modules/acquia_cms_image/composer.json [hunk 1]
```diff
@@ -1,5 +1,10 @@
 {
     "name": "drupal/acquia_cms_image",
     "type": "drupal-module",
-    "description": "Provides an Image media type and related configuration."
+    "description": "Provides an Image media type and related configuration.",
+    "require": {
+        "drupal/acquia_cms_common": "^1",
+        "drupal/field_group": "^3",
+        "drupal/focal_point": "1.5"
```
Comment (kdruckman) [inline modules/acquia_cms_image/composer.json:8]: This module also depends on default_content.

**Review by kdruckman — CHANGES_REQUESTED** [review]
All the modules should depend on acquia_cms_common since we can't rely on the profile being installed.

File: modules/acquia_cms_image/composer.json (hunk 1 above)
Comment (saurabhtripathics) [reply modules/acquia_cms_image/composer.json:9]: fixed

**Comment by saurabhtripathics** [comment]
> All the modules should depend on acquia_cms_common since we can't rely on the profile being installed.

done

File: modules/acquia_cms_article/composer.json [hunk 2]
```diff
@@ -1,5 +1,10 @@
 {
     "name": "drupal/acquia_cms_article",
     "type": "drupal-module",
-    "description": "Provides an Article content type and related configuration."
+    "description": "Provides an Article content type and related configuration.",
+    "require": {
+        "drupal/acquia_cms_image": "^1",
+        "drupal/acquia_cms_person": "^1",
+        "drupal/acquia_search": "^3.0"
```
[the remark this answers is not in the document]
Comment (saurabhtripathics) [reply modules/acquia_cms_article/composer.json]: removed drupal/acquia_search from article and added drupal/acquia_cms_article in search

**Pull request merged**